lib/plugins.ts

```diff
@@ -162,13 +162,13 @@ export interface ProtractorPlugin {
    * @throws {*} If this function throws an error, a failed assertion is added to
    *     the test results.
    *
-   * @return {Q.Promise<boolean>|boolean} If truthy, Protractor will continue onto
-   *     the next command.  If falsy, webdriver will continuously re-run this
-   *     function until it is truthy.  If a rejected promise is returned, a failed
-   *     assertion is added to the test results, and protractor will continue onto
-   *     the next command.
+   * @return {webdriver.promise.Promise<boolean>|Q.Promise<boolean>|boolean} If truthy,
+   *     Protractor will continue onto the next command.  If falsy, webdriver will
+   *     continuously re-run this function until it is truthy.  If a rejected promise
+   *     is returned, a failed assertion is added to the test results, and Protractor
+   *     will continue onto the next command.
    */
-  waitForCondition?: () => q.Promise<any>;
+  waitForCondition?(): webdriver.promise.Promise<boolean>|q.Promise<boolean>|boolean;
```

Comment:
`q.Promise` is actually not allowed here - only webdriver's promise implementation.  Could you fix the type/comment?